pipes/WDL/tasks/tasks_demux.wdl

```diff
@@ -292,6 +292,12 @@ task illumina_demux {
 
     illumina.py guess_barcodes --expected_assigned_fraction=0 barcodes.txt metrics.txt barcodes_outliers.txt
 
+    illumina.py flowcell_metadata --inDir $FLOWCELL_DIR flowcellMetadataFile.tsv
+
+    # output machine model and lane count
+    grep "machine" flowcellMetadataFile.tsv | cut -f2 | tee MACHINE_MODEL
+    grep "lane_count" flowcellMetadataFile.tsv | cut -f2 | tee LANE_COUNT
+
```

Comment:
I think you can drop these lines if you make the changes below in the output section